alma/alma-installments-prestashop
# Feat/mpp 89 basket items in payment only for credit

### Reason for change

We send the cart in payload payment only for credit (>4x) + Unit test for getCartItem

[Linear task](https://linear.app/almapay/issue/MPP-89/basket-items-in-payment-only-for-credit)

### Code changes

In payload for payment we add condition if installment > 4 + Unit test for getCartItem

### How to test

_As a reviewer, you are encouraged to test the PR locally._

Make a payment with multi product in cart, with virtual product in cart, with gift product in cart. 
Same test for > 4 installment and < 4 installment and check if cart is sent in dashboard Alma for installment > 4

### Checklist for authors and reviewers




### Non applicable
